app/Controllers/AddressChange/ShowUpdateAddressController.php

```diff
@@ -41,4 +52,12 @@ public function index( Request $request, FunFunFactory $ffFactory ): Response {
 			)
 		);
 	}
+
+	private function addressChangeIsAlreadyUpdated( string $addressToken, AddressChange $addressChange ): bool {
```

Comment:
This should be updated with the changes from https://github.com/wmde/fundraising-address-change/pull/52